runtime/etc/init/opa.conf

```diff
@@ -0,0 +1,27 @@
+description "Open Policy Agent"
+
+stop on shutdown
+
+respawn
+
+script
+      eval $(/opt/taupage/bin/parse-yaml.py /meta/taupage.yaml "config")
+      export AVAIL_ZONE="$(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone)"
```

Comment:
This should also `--fail`, we've had some issues with IMDS availability in the past.

Reply:
done